src/common/http-request/internal-api-headers.ts

```diff
@@ -1 +1,3 @@
 export const INTERNAL_USE_ONLY = 'This API endpoint is for internal use only and may change in the future';
+
+export const SDK_VERSION_HEADERS = { 'X-Checkout-SDK-Version': LIBRARY_VERSION };
```

Comment:
Could we move this into a separate file instead? This is because the header applies to public API as well, not just internal API.

Reply:
Yes, of course. I was assuming this file is for internal use headers😅.